### DISCUSSION TITLE
Have option to disable "Are you sure?" on closing the app

### DISCUSSION THREAD

**Discussion by SteveGoodenough** [cstayyab/WALC#208] [Feature Requests] [answered]
It's just annoying to have to confirm that I'm closing the app, either have the option to disable the prompt or just don't ask. Not many apps that do this anymore. Cheers

**Comment by daffamumtaz2361** [comment, marked as answer]
There's already an option to do that! Go to Dashboard > Settings > General > Ask on Exit
